bgs/bgs.go

```diff
@@ -460,6 +460,9 @@ func (bgs *BGS) EventsHandler(c echo.Context) error {
 		return fmt.Errorf("upgrading websocket: %w", err)
 	}
 
+	lastWriteLk := sync.RWMutex{}
```

Comment:
i'd just use a normal Mutex here, you only go faster with an RWMutex if reads are far more common than writes